test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -11,6 +11,23 @@
 public class UtilsTest {
 
 
+    @Test
+    public void elementsAreNull() throws Exception {
```

Comment:
Incorrect name for tests. Tests should be named in this format : `FeatureUnderTest_scenarioBeingTested_expectedOutcome()`
So it should at least have the name `isAnyNull` to show that it's testing that feature/method.